enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/iverksett/IverksettDto.kt

```diff
@@ -49,6 +50,7 @@ data class BehandlingsdetaljerDto(
 data class VedtaksdetaljerDto(
         val resultat: Vedtaksresultat,
         val vedtaksdato: LocalDate,
+        val vedtakstidspunkt: LocalDateTime,
```

Comment:
Denne må kanskje være nullable med default null for at vi ikke skal gå på en smell i prod?